### DISCUSSION TITLE
Pronterface

### DISCUSSION THREAD

**Discussion by Teobhusa** [Desuuuu/Marlin#22] [General]
[Q] Hi I can't not connect anymore with the Pronterface from USB connection on my computer after I flash the firmware! Everything work on the printer but no more connection to the computer via usb, what can I do to fix that if you can help me please?
PS. My printer is a Ender 5 plus.

**Comment by Desuuuu** [comment]
Make sure you select the correct baudrate when trying to connect. The default baudrate for this firmware is 250000.

There isn't anything else I can think of given so little information.
